tests/virt_autotest/guest_upgrade_run.pm

```diff
@@ -101,7 +101,12 @@ sub run {
     repl_module_in_sourcefile();
 
     $self->{'package_name'} = 'Guest Upgrade Test';
-    $self->execute_script_run("[ -d /var/log/qa/ctcs2 ] && rm -r /var/log/qa/ctcs2 ; [ -d /tmp/prj4_guest_upgrade ] && rm -r /tmp/prj4_guest_upgrade", 30);
+    if (check_var('ARCH', 's390x')) {
+        script_run "[ -d /var/log/qa/ctcs2 ] && rm -r /var/log/qa/ctcs2 ; [ -d /tmp/prj4_guest_upgrade ] && rm -r /tmp/prj4_guest_upgrade";
+    }
+    else {
+        $self->execute_script_run("[ -d /var/log/qa/ctcs2 ] && rm -r /var/log/qa/ctcs2 ; [ -d /tmp/prj4_guest_upgrade ] && rm -r /tmp/prj4_guest_upgrade", 30);
```

Comment:
Why you still use execute_script_run ?  I think you should change to use assert_script_run or script_run.

Reply:
yep, good point here. Because of  `assert_script_run()` do not support on s390x LPAR - OSD SUT. 
mean that we can not use with generally api, such as `assert_script_run` to trigger commands on OSD SUT s390x LPAR at all.
So, we have to call some special api, such as `lpar_cmd()` trigger command and ensure command feedback together or `script_run()` to just only trigger command without any ensured on OSD SUT s390x LPAR. 
Here, just use with `script_run()` to cleanup the existed /var/log/qa/ctcs2 and /tmp/prj4_guest_upgrade dirs on s390x LPAR.

Reply:
Yes. I should know this. s390x rules here.